### PR TITLE
[Enhancement](partial update) Add partial update mix cases

### PR DESCRIPTION
**Pull request by Yukang-Lian**
## Proposed changes

Issue Number: close #xxx

### CODE REVIEW

**Comment by Yukang-Lian** [comment]
run buildall

**Review by dataroaring — DISMISSED** [review]
LGTM

File: regression-test/suites/unique_with_mow_p0/partial_update/test_mix_partial_update.groovy [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+            // sql "insert into ${tableStreamName2} (k1,c2,c3,c4) values(2,2,2,{'a':200,'b':200})"
+            streamLoad {
+                table "${tableStreamName2}"
+                set 'columns', 'k1,c2,c3,c4'
+                set 'partial_columns', 'true'
+                set 'format', 'csv'
+                file "test_mix_partial_update_load_B.csv"
+                time 10000 // limit inflight 10s
+            }
+            qt_select_B "select k1,c1,c2,c3,c4 from ${tableStreamName2}"
```
Comment (zhannngchen) [inline regression-test/suites/unique_with_mow_p0/partial_update/test_mix_partial_update.groovy:327]: c1 is auto_increment column, it's value might be different with the output you expect when this case is running on cluster, in that condition, the case might fail.

**Comment by Yukang-Lian** [comment]
run buildall

**Review by zhannngchen — APPROVED** [review]
LGTM

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**